library/pages/components/GettingStartedPage/index.js

```diff
@@ -0,0 +1,144 @@
+                        className="react-rainbow-getting-started_top-container-tab" />
+                    <Tab
+                        name="contribuiting"
+                        label="CONTRIBUITING"
+                        className="react-rainbow-getting-started_top-container-tab react-rainbow-getting-started_top-container-tab_shortened" />
+                </Tabset>
+                <section className="react-rainbow-getting-started_container">
+                    <RenderIf isTrue={activeTabName === 'overview'}>
+                        <h3 className="section-heading">
+                            React Rainbow is a collection of components that will
+                            reliably help you build your application in a snap.
+                            Give it a hack and let us know what you think.
+                        </h3>
+                        <FeatureList />
+                    </RenderIf>
+                    <RenderIf isTrue={activeTabName === 'installation'}>
+                        <div className="rainbow-flex rainbow-flex_column">
+                            <h3 className="section-heading">
+                                React Rainbow Components is currently optimized for React 16.4.2
+                            </h3>
+                            <h2 className="section-heading-2">Install</h2>
+                            <h5 className="section-heading-5">
+                                React Rainbow Components is available as an
+                                <a
+                                    className="section-link"
+                                    href="https://www.npmjs.com/package/react-rainbow-components"
+                                    rel="noopener noreferrer"
+                                    target="_blank">
+                                    npm package
+                                </a>
+                            </h5>
+                            <span className="section-italic">installing with npm</span>
+                            <CodeEditor code="$ npm install react-rainbow-components --save" />
+                            <span className="section-italic">installing with yarn</span>
+                            <CodeEditor code="$ yarn add react-rainbow-components" />
+                        </div>
+                    </RenderIf>
+                    <RenderIf isTrue={activeTabName === 'usage'}>
+                        <h3 className="section-heading">
+                            There are several runnable examples in this Git repo,
+                            but here's a Hello World one:
+                        </h3>
+                        <CodeEditor code={HelloWorldExample} />
+                        <iframe
+                            title="codesandbox example"
+                            src="https://codesandbox.io/embed/24p8n0pnz0?hidenavigation=1&fontsize=14&view=preview"
+                            style={this.sampleStyles}
```

Comment:
why not use className? if somewhere a className is not possible then use style.js file. Remember separation of concerns.